lib/awskeyring/awsapi.rb

```diff
@@ -185,7 +185,7 @@ def self.region
     # @return [String] key The aws_access_key_id
     # @return [String] secret The aws_secret_access_key
     # @return [String] account the associated account name.
-    def self.rotate(account:, key:, secret:, key_message:) # rubocop:disable  Metrics/MethodLength, Metrics/AbcSize
+    def self.rotate(account:, key:, secret:, key_message:) # rubocop:disable  Metrics/MethodLength
```

Comment:
Please add comments around motivations for ignoring rubocop directives.

Reply:
I am trying to have the minimal exceptions to the default RuboCop rules. this exception was no longer needed.

Reply:
Sorry, I misread here, I thought you were adding an ignore rule. All good.